enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -24,66 +32,29 @@ use alloc::{
 	string::String,
 	vec::Vec,
 };
-use core::{ops::Deref, result::Result};
-
-use sgx_types::*;
-use std::sync::{
-	atomic::{AtomicPtr, Ordering},
-	Arc, SgxMutex,
-};
-
-use sp_core::H256 as Hash;
-
+use base58::FromBase58;
 use codec::{Decode, Encode};
-use log::*;
-
-use crate::rpc::{
-	api::SideChainApi,
-	author::{Author, AuthorApi},
-	basic_pool::BasicPool,
-};
-
-use crate::top_pool::pool::Options as PoolOptions;
-
-use self::serde_json::*;
-use jsonrpc_core::{futures::executor, Error as RpcError, *};
-
+use core::{ops::Deref, result::Result};
 use ita_stf::ShardIdentifier;
-
-use base58::FromBase58;
-
 use itp_types::{
-	block::SignedBlock, Block, DirectRequestStatus, Request, RpcReturnValue, TrustedOperationStatus,
+	block::SignedBlock, DirectRequestStatus, Request, RpcReturnValue, TrustedOperationStatus,
 };
+use jsonrpc_core::{futures::executor, Error as RpcError, *};
+use log::*;
+use sgx_types::*;
+use sp_core::H256 as Hash;
+use std::sync::Arc;
 
-use crate::{rsa3072, utils::write_slice_and_whitespace_pad, OcallApi};
-
-static GLOBAL_TX_POOL: AtomicPtr<()> = AtomicPtr::new(0 as *mut ());
```

Comment:
remove this global variable. access to the trusted operation pool is done via the `GetTopPool` trait and its global implementation `GlobalTopPoolContainer` (or for tests, the local one: `TopPoolContainer<TopPool>`